[OS X][DUB] OS X python2 uses UCS2 version strings.

Failed to build [simple dub example](https://github.com/9il/atmosphere_gm_charts).
```
Building pyd ~master configuration "python27", build type debug.
Running dmd...
Building atmosphere_gm_charts ~master configuration "application", build type debug.
Compiling using dmd...
Linking...
Undefined symbols for architecture x86_64:
  "_PyUnicodeUCS4_AsUTF8String", referenced from:
      _D3pyd11make_object28__T18python_to_d_stringTAyaZ18python_to_d_stringFPS6deimos6python6object8PyObjectZAya in libpyd.a(make_object_c73_a55.o)
      _D3pyd9exception17printGenericErrorFPS6deimos6python6object8PyObjectPS6deimos6python6object8PyObjectPS6deimos6python6object8PyObjectZAya in libpyd.a(exception_be1_6f4.o)
  "_PyUnicodeUCS4_DecodeUTF8", referenced from:
      _D3pyd11make_object28__T18d_string_to_pythonTAyaZ18d_string_to_pythonFAyaZPS6deimos6python6object8PyObject in atmosphere_gm_charts.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
--- errorlevel 1
FAIL .dub/build/application-debug-posix.osx-x86_64-dmd_2067-9B15B55A11B9D465A817671691F23DD2/ atmosphere_gm_charts executable
Error executing command run: dmd failed with exit code 1.
```

Linking:
```
dmd -c -of.dub/build/application-debug-posix.osx-x86_64-dmd_2067-5A034B66DAC606C3055DA18D77ACAB68/atmosphere_gm_charts.o -debug -g -w -version=Python_2_4_Or_Later -version=Python_2_5_Or_Later -version=Python_2_6_Or_Later -version=Python_2_7_Or_Later -version=Python_Unicode_UCS4 -version=Have_atmosphere_gm_charts -version=Have_pyd -Isource/ -I../../.dub/packages/pyd-master/infrastructure/ source/app.d
Linking...
dmd -of.dub/build/application-debug-posix.osx-x86_64-dmd_2067-5A034B66DAC606C3055DA18D77ACAB68/atmosphere_gm_charts .dub/build/application-debug-posix.osx-x86_64-dmd_2067-5A034B66DAC606C3055DA18D77ACAB68/atmosphere_gm_charts.o ../../.dub/packages/pyd-master/libpyd.a -L-v -L-lpython2.7 -g
@(#)PROGRAM:ld  PROJECT:ld64-241.9
configured to support archs: armv6 armv7 armv7s arm64 i386 x86_64 x86_64h armv6m armv7m armv7em
Library search paths:
	/usr/share/dmd/lib
	/usr/lib
	/usr/local/lib
Framework search paths:
	/Library/Frameworks/
	/System/Library/Frameworks/
Undefined symbols for architecture x86_64:
  "_PyUnicodeUCS4_AsUTF8String", referenced from:
      _D3pyd11make_object28__T18python_to_d_stringTAyaZ18python_to_d_stringFPS6deimos6python6object8PyObjectZAya in libpyd.a(make_object_c73_a55.o)
      _D3pyd9exception17printGenericErrorFPS6deimos6python6object8PyObjectPS6deimos6python6object8PyObjectPS6deimos6python6object8PyObjectZAya in libpyd.a(exception_be1_6f4.o)
  "_PyUnicodeUCS4_DecodeUTF8", referenced from:
      _D3pyd11make_object28__T18d_string_to_pythonTAyaZ18d_string_to_pythonFAyaZPS6deimos6python6object8PyObject in atmosphere_gm_charts.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
--- errorlevel 1

```

--- a/dub.json
+++ b/dub.json
@@ -42,7 +42,7 @@
                 "Python_3_3_Or_Later",
             ],
             "libs": ["python3.3m"],
-            "lflags": ["-L/Library/Frameworks/Python.framework/Versions/3.3/lib/"]
+            "lflags-osx": ["-L/Library/Frameworks/Python.framework/Versions/3.3/lib/"]
         },
         {
             "name": "python34",
@@ -58,7 +58,7 @@
                 "Python_3_4_Or_Later",
             ],
             "libs": ["python3.4m"],
-            "lflags": ["-L/Library/Frameworks/Python.framework/Versions/3.4/lib/"]
+            "lflags-osx": ["-L/Library/Frameworks/Python.framework/Versions/3.4/lib/"]
         }
     ]
 }

--- a/dub.json
+++ b/dub.json
@@ -2,6 +2,7 @@
     "name": "pyd",
     "description": "Pyd is a library for the D programming language that wraps the raw Python/C API with a cleaner, simpler interface.",
     "homepage": "https://github.com/ariovistus/pyd",
+    "targetType": "sourceLibrary",
     "authors": [
         "Kirk McDonald"
     ],

--- a/dub.json
+++ b/dub.json
@@ -15,50 +15,50 @@
         "infrastructure/meta/",
         "infrastructure/util/"
     ],
+    "versions-linux":   ["Python_Unicode_UCS4"],
+    "versions-osx":     ["Python_Unicode_UCS2"],
+    "versions-windows": ["Python_Unicode_UCS2"],
     "configurations": [
-    {
-        "name": "python27", 
-        "versions": [
-            "Python_2_4_Or_Later", 
-            "Python_2_5_Or_Later", 
-            "Python_2_6_Or_Later", 
-            "Python_2_7_Or_Later", 
-            "Python_Unicode_UCS4"
-        ], 
-        "libs": [
-            "python2.7"
-        ]
-    },
-    {
-        "name": "python27-shared", 
-        "targetType": "dynamicLibrary",
-        "dflags": ["-fPIC", "-defaultlib=libphobos2.so"],
-        "versions": [
-            "Python_2_4_Or_Later", 
-            "Python_2_5_Or_Later", 
-            "Python_2_6_Or_Later", 
-            "Python_2_7_Or_Later", 
-            "Python_Unicode_UCS4"
-        ], 
-        "libs": [
-            "python2.7"
-        ]
-    },
-    {
-        "name": "python33", 
-        "versions": [
-            "Python_2_4_Or_Later", 
-            "Python_2_5_Or_Later", 
-            "Python_2_6_Or_Later", 
-            "Python_2_7_Or_Later", 
-            "Python_3_0_Or_Later", 
-            "Python_3_1_Or_Later", 
-            "Python_3_2_Or_Later", 
-            "Python_3_3_Or_Later", 
-            "Python_Unicode_UCS4"
-        ], 
-        "libs": [
-            "python3.3m"
-        ]
-    }]
+        {
+            "name": "python27",
+            "versions": [
+                "Python_2_4_Or_Later",
+                "Python_2_5_Or_Later",
+                "Python_2_6_Or_Later",
+                "Python_2_7_Or_Later",
+            ],
+            "libs": ["python2.7"]
+        },
+        {
+            "name": "python33",
+            "versions": [
+                "Python_2_4_Or_Later",
+                "Python_2_5_Or_Later",
+                "Python_2_6_Or_Later",
+                "Python_2_7_Or_Later",
+                "Python_3_0_Or_Later",
+                "Python_3_1_Or_Later",
+                "Python_3_2_Or_Later",
+                "Python_3_3_Or_Later",
+            ],
+            "libs": ["python3.3m"],
+            "lflags": ["-L/Library/Frameworks/Python.framework/Versions/3.3/lib/"]
+        },
+        {
+            "name": "python34",
+            "versions": [
+                "Python_2_4_Or_Later",
+                "Python_2_5_Or_Later",
+                "Python_2_6_Or_Later",
+                "Python_2_7_Or_Later",
+                "Python_3_0_Or_Later",
+                "Python_3_1_Or_Later",
+                "Python_3_2_Or_Later",
+                "Python_3_3_Or_Later",
+                "Python_3_4_Or_Later",
+            ],
+            "libs": ["python3.4m"],
+            "lflags": ["-L/Library/Frameworks/Python.framework/Versions/3.4/lib/"]
+        }
+    ]
 }

